src/common/hooks/confirm.ts

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+
+export type ConfirmProps = {
+  onConfirm?: () => void | Promise<void>
+  onClose?: () => void | Promise<void>
+}
+export type ConfirmHookProps = {
+  open: boolean
+  loading: boolean
+  openHandler: () => void
+  confirmHandler: () => void
+  closeHandler: () => void
+}
+
+const useConfirm = ({ onConfirm, onClose }: ConfirmProps): ConfirmHookProps => {
```

Comment:
Awesome!

 Let's rename the filename to match the hook name - `useConfirm.ts`

Reply:
Done in [95fedb9](https://github.com/podkrepi-bg/frontend/pull/728/commits/95fedb9d7a1c818ab2b7b06e662d78ad23679d4e)